at_onboarding_flutter/lib/screens/pair_atsign.dart

```diff
@@ -773,7 +779,9 @@ class _PairAtsignWidgetState extends State<PairAtsignWidget> {
             builder: (_) => WillPopScope(
               onWillPop: () async {
                 int ct = 0;
-                Navigator.of(context).popUntil((_) => ct++ >= 2);
+                Navigator.of(context).popUntil((_) {
+                  return ct++ >= 2;
```

Comment:
Can you explain why this is? And if not necessary, please remove it.

Reply:
I don't completely understand it's behaviour, haven;t made any changes to it. Just converted it from the arrow function to block function.

Reply:
Yes, we need to provide a false statement there. We can either pass false there.